frontend/webpack.config.ts

```diff
@@ -39,7 +36,7 @@ const config: webpack.Configuration = {
       { test: /\.glsl$/, loader: 'raw!glslify' },
       {
         test: /(\.jsx?)|(\.tsx?)$/,
-        exclude: /node_modules/,
+        exclude: /node_modules\/(?!(@console)\/)/,
```

Comment:
Are we sure this is right?

Reply:
I'm sure this is the magic line. The current `exclude` excludes everything since all `@console/*` references actually are in `node_modules/@console/*`.

Reply:
> I'm sure this is the magic line. The current `exclude` excludes everything since all `@console/*` references actually are in `node_modules/@console/*`.

I'm still not sure. If I only bump kubevirt-web-ui-components with no other changes, I get a main vendor bundle size of 7.52MB, which is the same as https://github.com/openshift/console/pull/2232#issuecomment-522080994